### ISSUE TITLE
Calendar icon placed on the top of another

### ISSUE THREAD

**Issue by EflammOllivier** [kekeh/mydatepicker#232]
Hello,

I use the version 1.7.9 of the module. I tested it with the latest versions of Opera and Firefox. I don't have this issue when i run your demo.  
There are two calendar icons that one is placed on the top of the other. Also the cross of the clear button is not centered. And the selector doesn't open when i click on the input, even with `openSelectorOnInputClick : true`.
![screenshot-localhost-3000-2017-03-23-14-24-07](https://cloud.githubusercontent.com/assets/5957955/24251111/5add0bfe-0fd9-11e7-99cd-b2371c9d1392.png)
This is how I integrated the datepicker in my template,
```html
<my-date-picker [options]="myDatePickerOptions" (dateChanged)="onDateChanged($event.jsdate)"></my-date-picker>
```
These are the option that I have used,
```javascript
private myDatePickerOptions: IMyOptions = {
        dayLabels : {su: 'Dim', mo: 'Lun', tu: 'Mar', we: 'Mer', th: 'Jeu', fr: 'Ven', sa: 'Sam'}, 
        monthLabels : { 1: 'Jan', 2: 'Fév', 3: 'Mar', 4: 'Avr', 5: 'Mai', 6: 'Juin', 7: 'Juil', 8: 'Août', 9: 'Sept', 10: 'Oct', 11: 'Nov', 12: 'Déc' },    
        todayBtnTxt : "Aujourd'hui", 
        minYear : 2000, 
        dateFormat: 'dd/mm/yyyy',
        showClearDateBtn : true,
        openSelectorOnInputClick : true, 
    };
```
I manage to resolve the centering problem deleting the following part, using the browser debug tool,
```css
element.style {  
    line-height: 34px;   
}
```
And to remove one the icons, I deleted this part,
```css
.mydp .icon-calendar:before {`    
    `content: "\e802";`  
}
```
Is that a bug, or did I make a mistake while integrating the module ?

**Comment by kekeh** [comment]
Are you using some other javascript library which supports font icons? And is there font icon named **icon-calendar**?

Calendar icon in the mydatepicker template:
```html
<button type="button" [attr.aria-label]="opts.ariaLabelOpenCalendar" class="btnpicker" (click)="openBtnClicked()" [ngClass]="{'btnpickerenabled': !opts.componentDisabled, 'btnpickerdisabled': opts.componentDisabled, 'btnleftborder': opts.showInputField||selectionDayTxt.length>0&&opts.showClearDateBtn}" [disabled]="opts.componentDisabled">
    <span class="mydpicon icon-calendar" [ngStyle]="{'line-height': opts.height}"></span>
</button>
```

Font icon data is end of this file: https://github.com/kekeh/mydatepicker/blob/master/src/my-date-picker/my-date-picker.component.css

**Comment by kekeh** [comment]
I did one change. Try version 1.7.10. Let me know if it is working or not.

**Comment by EflammOllivier** [comment]
Yes it worked, thanks. But the icon is still not centered.
![screenshot-localhost-3000-2017-03-23-14-24-07](https://cloud.githubusercontent.com/assets/5957955/24295994/86c01a12-109d-11e7-91e5-d0f08586d2e3.png)
As I said before, the following CSS block is the cause. I found this using the browser debug tool.
```css
element.style {
    line-height: 34px;
}
```

**Comment by kekeh** [comment]
Yes, it was unnecessary definition and I removed the line-height definition from the font icon element. 

You can now try fixed version 1.7.13.

**Comment by EflammOllivier** [comment]
Yes it all worked out, thanks.
